torchx/workspace/docker_workspace.py

```diff
@@ -166,13 +167,42 @@ def push_images(self, images_to_push: Dict[str, Tuple[str, str]]) -> None:
             log.info(f"pushing image {repo}:{tag}...")
             img = client.images.get(local)
             img.tag(repo, tag=tag)
-            for line in client.images.push(repo, tag=tag, stream=True, decode=True):
+            ids = []
+            for event in client.images.push(repo, tag=tag, stream=True, decode=True):
```

Comment:
can we pull this printing logic out into a helper method that takes the iterator from push?

would also be nice to have a unit test

Reply:
Done. Pulled the print logic out to a function called `print_push_event()` and added unittest for it with a sample event file.